zng/docs/zeek-compat.md

```diff
@@ -1,138 +1,188 @@
 # ZNG Compatibility with Zeek Logs
 
-THIS SECTION IS A WORK IN PROGRESS.  WE DECIDED TO FOREGO COMPLETE
-FORMAT-COMPATIBILITY BETWEEN ZNG AND ZEEK AND WILL INSTEAD DOCUMENT HERE THE
-RELATIONSHIP BETWEEN ZEEK AND ZNG FORMAT.
+- [Introduction](#introduction)
+- [Equivalent Types](#equivalent-types)
+- [Example](#example)
+- [Type-Specific Details](#type-specific-details)
+  * [`double`](#double)
+  * [`set`](#set)
+  * [`enum`](#enum)
+  * [`string`](#string)
+  * [`record`](#record)
+
+## Introduction
+
+As the ZNG design was motivated by the [Zeek log format](https://docs.zeek.org/en/stable/examples/logs/),
+care has been taken to maintain compatibility with it. This document describes
+how the type system described in the [ZNG specification](spec.md)
+is able to represent each of the types that may appear in Zeek logs.
+
+Tools like [`zq`](https://github.com/brimsec/zq) and [Brim](https://github.com/brimsec/brim)
+maintain an internal ZNG representation of any Zeek data that is read or
+imported. Therefore, knowing the equivalent types will prove useful when
+performing [ZQL](../../zql/README.md) operations such as
+[type casting](../../zql/docs/data-types#example) or looking at the
+data when output as [TZNG](spec.md#4-zng-text-format-tzng).
+
+## Equivalent Types
+
+The following table summarizes which ZNG data type corresponds to each
+[Zeek data type](https://docs.zeek.org/en/current/script-reference/types.html)
+that may appear in a Zeek log. While most types have a simple 1-to-1 mapping
```

Comment:
I think it's worth warning that the linked page describes the Zeek scripting language types, which are a superset of the Zeek log file types with encodings that differ in some places, and that we refer to it because there is no authoritative specification of the Zeek log format.

Reply:
@nwt: Thanks for speaking up about this! It did cross my mind, so your comment pushed me over into adding some text. Feel free to give it a read and suggest adjustments. I intentionally opted to put the clarification _below_ the table so as to not interrupt flow.

Reply:
I'm going ahead and merging as-is. Happy to revisit in a future pass if you have any feedback later.